pages/api/revalidate.js

```diff
@@ -31,11 +34,14 @@ async function handler(req, res) {
   }
 
   try {
-    const searchParams = new URLSearchParams(query);
-
-    await fetch(`${BASE_URL}/api/app-revalidate?${searchParams.toString()}`);
-    await res.revalidate(`/${query.uri}`);
-    return res.status(200).json({ revalidated: true });
+    if (APP_ROUTER_REVALIDATE.includes(query.uri)) {
```

Comment:
try to minimize errors where revalidation occurs in both environments but one is missing data, this can be removed when migration is complete